admin/class-purger.php

```diff
@@ -682,42 +1064,77 @@ public function purge_them_all() {
 		$this->_purge_all_posts();
 		$this->_purge_all_taxonomies();
 		$this->_purge_all_date_archives();
-		$this->log( __( 'Everthing purged!', 'nginx-helper' ) );
+		$this->log( __( 'Everything purged!', 'nginx-helper' ) );
+
 		return true;
+
 	}
 
+	/**
+	 * Purge cache on term edited.
+	 *
+	 * @param int    $term_id Term id.
+	 * @param int    $tt_id Taxonomy id.
+	 * @param string $taxon Taxonomy.
+	 *
+	 * @return bool
+	 */
 	public function purge_on_term_taxonomy_edited( $term_id, $tt_id, $taxon ) {
 
 		$this->log( __( 'Term taxonomy edited or deleted', 'nginx-helper' ) );
 
-		if ( 'edit_term' == current_filter() && $term = get_term( $term_id, $taxon ) ) {
+		$term = get_term( $term_id, $taxon );
+
+		if ( 'edit_term' === current_filter() && ! is_wp_error( $term ) && ! empty( $term ) ) {
+
 			$this->log( sprintf( __( "Term taxonomy '%1\$s' edited, (tt_id '%2\$d', term_id '%3\$d', taxonomy '%4\$s')", 'nginx-helper' ), $term->name, $tt_id, $term_id, $taxon ) );
-		} elseif ( 'delete_term' == current_filter() ) {
+
+		} elseif ( 'delete_term' === current_filter() ) {
+
 			$this->log( sprintf( __( "A term taxonomy has been deleted from taxonomy '%1\$s', (tt_id '%2\$d', term_id '%3\$d')", 'nginx-helper' ), $taxon, $term_id, $tt_id ) );
+
 		}
 
 		$this->_purge_homepage();
 
 		return true;
+
 	}
 
+	/**
+	 * Check ajax referrer on purge.
+	 *
+	 * @param string $action Action.
+	 * @param string $result Result.
```

Comment:
Why you're adding comment as variable name only? This variable comment suppose to explain about variable usage in one line. And this are WP hooks you can easily find out comments in core. 
check this: https://github.com/WordPress/WordPress/blob/bfc53e6a52fa837f700c59d46911aab727985689/wp-includes/pluggable.php#L1164